tests/neg/trailingCommas/trailingCommas.scala

```diff
@@ -0,0 +1,58 @@
+package foo
+
+// Multi-line only cases: make sure trailing commas are only supported when multi-line
+
+trait ArgumentExprs1 { validMethod(23, "bar", )(Ev0, Ev1) } // error // error
```

Comment:
What are the two errors emitted here? One is the invalid trailing comma. What is the other one?

Reply:
The first one is `illegal start of simple expression`. The second is as follow
```
5 |trait ArgumentExprs1 { validMethod(23, "bar", )(Ev0, Ev1) } // error // error
  |                                            ^
  |too many arguments for method validMethod: (foo: Int, bar: String)(implicit ev0: foo.package.Ev0, ev1: foo.package.Ev1): Int
```